src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -456,7 +457,7 @@ def getClusterAd(self):
         ad['Cmd'] = self.scriptFile
         ad['Out'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".out")')
         ad['Err'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".err")')
-        ad['UserLog'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".log")')
+        ad['UserLog'] = classad.ExprTree('strcat(Iwd, "/condor.", ClusterId, ".", ProcId, ".log")')
```

Comment:
For simplicity of comprehension, it might be a good idea to either apply the `Iwd` prefix to all three of `Out`, `Err`, and `UserLog` or to none.